Changes.md

```diff
@@ -6,6 +6,15 @@ Fixes
 
 - DispatchDialogue : Fixed `AttributeError: '__Implementation' object has no attribute 'message'` error.
 
+Improvements
```

Comment:
The improvements section should be above the fixes section.

Reply:
Fixed in the rebased commits https://github.com/GafferHQ/gaffer/pull/5207/commits/de5c210937e610e676a13c2530e04f841fae8574 and https://github.com/GafferHQ/gaffer/pull/5207/commits/c6b8f793d94feea1526781c49fce9a56c172971c